library/include/hipblas.h

```diff
@@ -19873,6 +19945,9 @@ HIPBLAS_EXPORT hipblasStatus_t hipblasScalEx(hipblasHandle_t   handle,
 
     - Supported types are determined by the backend. See rocBLAS/cuBLAS documentation.
 
+    When compiling with HIPBLAS_V2, hipblasScalBatchedEx accepts hipDataType for alphaType,
```

Comment:
Is there a different include file hipblas_v2.h as well to force this?   or just -DHIPBLAS_V2 ?  Could say "HIPBLAS_V2 define" or similar.  I presume it matches rocblas methodology, so fine.

Reply:
Just HIPBLAS_V2, no hipblas_v2.h. Updated the comment a bit.